### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by Tuscan-blue**
Add a description of the steps for self-hosting `haven` using `docker-compose`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,11 @@ Fully managed hosting of your personal Haven is available too, check out: https:
 
 ## Docker
 
+1. Install `docker` and `docker-compose`
```
Comment (mawise) [inline README.md:74]: Let's add links for docker and docker-compose for further details on how to install them.

**Comment by mawise** [comment]
Thanks for helping improve the docs!!

File: README.md (hunk 1 above)
Comment (Tuscan-blue) [reply README.md]: Now It's OK!

**Comment by mawise** [comment]
Thank you!

**Pull request merged**